apps/mwiniimporter/importer.cpp

```diff
@@ -676,7 +676,7 @@ MwIniImporter::multistrmap MwIniImporter::loadIniFile(const boost::filesystem::p
         }
 
         if(line[0] == '[') {
-            int pos = line.find(']');
+            int pos = static_cast<int>(line.find(']'));
```

Comment:
Is there any advantage to casting to `int` instead of just using the `size_t`?

Reply:
Initially I used size_t, but it required to change conditions such as 
```
if(pos < 1)
```
to something like
```
if (pos == 0 || pos == std::string::npos)
```
in many places. Such changes do not make code to look any better, so I dropped them. If you think that size_t is preferable here, I can return it back.